Orange/widgets/data/owpivot.py

```diff
@@ -320,7 +320,11 @@ def map_values(index, _X):
                 continue
             if idx == 0:
                 self.renamed.append(self._row_var.name)
-                self._row_var = self._row_var.copy(name=u)
+                if self._row_var == self._col_var:
```

Comment:
In both cases you have `self._row_var = self._row_var.copy(name=u)`, so you can do this before `if`?

I suppose that you want to check that the variables are the same? Then you perhaps want to change `self._row_var == self._col_var` to `self._row_var is self._col_var`.